docs/ref/contrib/sites.txt

```diff
@@ -47,11 +47,11 @@ Why would you use sites? It's best explained through examples.
 Associating content with multiple sites
 ---------------------------------------
 
-The Django-powered sites LJWorld.com_ and Lawrence.com_ are operated by the
-same news organization -- the Lawrence Journal-World newspaper in Lawrence,
-Kansas. LJWorld.com focuses on news, while Lawrence.com focuses on local
-entertainment. But sometimes editors want to publish an article on *both*
-sites.
+Historically, the Django-powered sites LJWorld.com and Lawrence.com were
```

Comment:
I think that `LJWorld.com` and `Lawrence.com` description is still valid, the only difference is that they don't use Django anymore, so maybe :thinking: :
```
The LJWorld.com_ and Lawrence.com_ sites are operated by the same news
organization -- the Lawrence Journal-World newspaper in Lawrence, Kansas.
LJWorld.com focused on news, while Lawrence.com focused on local entertainment.
But sometimes editors wanted to publish an article on *both* sites.
```

I cannot check `ljword.com` :man_facepalming: :man_shrugging: so I'm not sure: 

![obraz](https://user-images.githubusercontent.com/2865885/70054293-8adb4800-15d7-11ea-8241-5200edc37892.png)

Reply:
One day the US will get better at user privacy -- hopefully. 😿